rstudio/rstudio
# remove inert polyfill

### Intent

Addresses #14353

### Approach

The "inert" polyfill is no longer doing anything as all supported browsers have natively supported the `inert` attribute for some time. https://caniuse.com/?search=inert

### Automated Tests

None

### QA Notes

The polyfill has been a no-op for years; everything in it is bypassed if the browser supports `inert` natively, but we were still loading it into the page. So the risk of this change is extremely low.

To test this:

- use RStudio Server (can't hit this scenario in Desktop)
- display a modal dialog such as Global Options
- put keyboard focus outside the page, e.g. in the browser's address bar
- use Tab key to move forward until focus goes to RStudio
- or use Shift+tab to move backwards until focus goes to RStudio

Working: focus jumps directly to the modal dialog
Not Working: focus goes to the page behind the modal dialog, e.g. the "File" menu

It is "inert" that prevents this from happening. We put it on the main part of the page when a modal dialog is displayed. When modal dialog are stacked, we put inert on the dialogs behind the topmost one.
 
### Documentation

None, not including in NEWS.

### Checklist
